telephone_appointments.gemspec

```diff
@@ -26,4 +26,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop', '~> 0.41'
+  spec.add_development_dependency 'vcr', '~> 3.0'
+  spec.add_development_dependency 'webmock', '~> 2.1'
```

Comment:
It's not in this diff but you don't need to reference `GlobalID`. Also these are pessimistically pinned at old versions. We should get them up to date as far as possible first.